firebase-perf/src/main/java/com/google/firebase/perf/util/Rate.java

```diff
@@ -0,0 +1,64 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.perf.util;
+
+import java.util.concurrent.TimeUnit;
+
+/** A Rate object representing the number of tokens per total specified time unit. */
+public class Rate {
+
+  private static final long NANO_IN_SECOND = 1000 * 1000 * 1000;
```

Comment:
Avoid doing maths on times. It's dangerous and standard libraries will do the conversions better than we can, so it's safer to use them.

Reply:
Fixed by using the standard Timeunit library.